themes/red-theme/lms/templates/footer.html

```diff
@@ -67,27 +69,26 @@
 
       ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
       <p class="copyright">
-        ## Translators: 'EdX', 'edX', and 'Open edX' are trademarks of 'edX Inc.'. Please do not translate any of these trademarks and company names.
-        ${_("EdX, Open edX, and the edX and Open edX logos are registered trademarks or trademarks of {link_start}edX Inc.{link_end}").format(
-            link_start=u"<a href='https://www.edx.org/'>",
-            link_end=u"</a>"
+        ## Translators: 'edX' and 'Open edX' are trademarks of 'edX Inc.'. Please do not translate any of these trademarks and company names.
+        ${Text(_("edX, Open edX, and the edX and Open edX logos are registered trademarks of {link_start}edX Inc.{link_end}")).format(
+            link_start=HTML(u"<a href='https://www.edx.org/'>"),
+            link_end=HTML(u"</a>")
         )}
       </p>
       <nav class="nav-legal" aria-label="${_('Legal')}">
         <ul>
         % if marketing_link('HONOR') and marketing_link('HONOR') != '#':
           <li class="nav-legal-01">
             <%
-              tos_link = u"<a href='{}'>".format(marketing_link('TOS'))
-              honor_link = u"<a href='{}'>".format(marketing_link('HONOR'))
+              tos_link = HTML(u"<a href='{}'>").format(HTML(marketing_link('TOS')))
+              honor_link = HTML(u"<a href='{}'>").format(HTML(marketing_link('HONOR')))
             %>
             ${
-              _(
-                "{tos_link_start}Terms of Service{tos_link_end} and {honor_link_start}Honor Code{honor_link_end}").format(
-                tos_link_start=tos_link,
-                tos_link_end="</a>",
-                honor_link_start=honor_link,
-                honor_link_end="</a>",
+              Text(_("{tos_link_start}Terms of Service{tos_link_end} and {honor_link_start}Honor Code{honor_link_end}")).format(
+                tos_link_start=HTML(tos_link),
```

Comment:
Nits: 
1. Why not name `tos_link` as `tos_link_start` since that name is more precise anyway?
2. Note: you don't need this `HTML()` call because `tos_link` is already HTML.  It's also fine to have it for consistency, which may be what you wanted?